sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -11,8 +11,8 @@
   "paths": {
     "/configurations/{id}": {
       "get": {
-        "summary": "Retrieve a configuration for Iot Hub devices and modules by it identifier.",
-        "description": " For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "summary": "Retrieve a configuration for Iot Hub device or module.",
+        "description": "Retrieve a configuration for Iot Hub device or modules.",
```

Comment:
The summary and description here are redundant. When we generate code, only the description is used anywhere. I've been just deleting these summary tags when they don't have anything to say beyond what the description already says

Reply:
Yeah there are lots of those and I saw that only description is used during generation but did not delete to keep consistency. But if you are deleting them all over the place, i'll go ahead and do it too.